src/chigame/users/urls.py

```diff
@@ -26,4 +27,8 @@
     path("accept_friend_invitation/<int:pk>", view=accept_friend_invitation, name="accept-friend-invitation"),
     path("decline_friend_invitation/<int:pk>", view=decline_friend_invitation, name="decline-friend-invitation"),
     path("user_history/<int:pk>", views.user_history, name="user-history"),
+    path("password_reset/", auth_views.PasswordResetView.as_view(), name="password_reset"),
```

Comment:
I can't seem to reset my password for any test accounts (i.e. user1@example.com) because it brings me to a fake page. This probably has something to do with the Django auth views, but do you know if there's any way to fix this?